src/base/system_parameter.h

```diff
@@ -539,13 +677,6 @@ extern "C"
 
   extern void *prm_get_value (PARAM_ID prm_id);
```

Comment:
By your modification, prm_get_value is called in only case of SERVER_MODE. Therefore, you should modifity "prm_get_value" to reflect your modified logic.

Reply:
There may be cases where only prm_get_value () is called separately, so I did not fix it.

Reply:
The "prm_get_value" is called from "css_sockout" and the other calls are only from system_param local funcions as like "prm_get_xxxx_value". The "css_sockout" is modified simply as calling "prm_get_integer_value". In my opinion, the "prm_get_value" should be modified because it has redundant logics as assert (prm_id <= PRM_LAST_ID) and "PRM_SERVER_SESSION" checking.